lms/djangoapps/onboarding/email_utils.py

```diff
@@ -61,13 +61,27 @@ def send_admin_update_email(org_id, org_name, dest_addr, hash_key, claimed_by_em
     MandrillClient().send_mail(MandrillClient.ORG_ADMIN_CHANGE_TEMPLATE, dest_addr, message_context)
 
 
-def send_admin_update_confirmation_email(org_name, dest_addr, confirm):
+def send_admin_update_confirmation_email(org_name, current_admin, new_admin, confirm):
     """
     Send an email to the claimed admin, that he is either accepted as admin or rejected
     """
```

Comment:
please add some description for parameters

Reply:
done.